templates/microcloud.html

```diff
@@ -0,0 +1,249 @@
+                        <div class="p-image-container p-section--shallow">
+                            <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/c459b7cd-microcloud-visual-1.svg" alt="" width="100">
+                        </div>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <h3 class="p-heading--5">自動デプロイ</h3>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <p>MicroCloudはシンプルさと自動化を重視して開発されました。MicroCloudでは、デプロイもオンサイトの運用も自動化され、オフィスの裏でも遠く離れたエッジサイトでも、オンサイトでのメンテナンスはほとんど、またはまったく必要ありません。
+                        </p>
+                    </div>
+                </div>
+                <div class="p-equal-height-row__col">
+                    <div class="p-equal-height-row__item">
+                        <div class="p-image-container p-section--shallow">
+                            <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/273d81f3-microcloud-visual-2.svg" alt="" width="90">
+                        </div>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <h3 class="p-heading--5">自動化された更新でセキュリティを確保</h3>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <p>たとえ小さくてもMicroCloudは大量のデータを処理します。機密性の高いワークロードを実行することもあります。データの安全を守るには、セキュリティと脆弱性に即座に対応する更新が優先事項です。MicroCloudのコンポーネントはsnapでパッケージ化されているため、隔離した状態で安心して更新できます
+                        </p>
+                    </div>
+                </div>
+                <div class="p-equal-height-row__col">
+                    <div class="p-equal-height-row__item">
+                        <div class="p-image-container p-section--shallow">
+                            <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/a8264def-microcloud-visual-3.svg" alt="" width="100">
+                        </div>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <h3 class="p-heading--5">コストとリソースを効率的に利用</h3>
+                    </div>
+                    <div class="p-equal-height-row__item">
+                        <p>標準的なUbuntu ServerとDesktopに加え、MicroCloudは、エッジに最適化した軽量OSのUbuntu Coreでも実行できます。Ubuntu
+                            Coreを使えば非常に軽量のクラウドが実現し、パフォーマンスで妥協することなくリソースを効率的に利用し、コストを抑制できます。</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+        <hr />
+        <div class="col">
+            <h2>MicroCloudのサイズ</h2>
+        </div>
+        <div class="col p-section--shallow">
+            <div class="u-embedded-media">
+                <iframe title="What is a MicroCloud?" class="u-embedded-media__element"
+                    src="https://www.youtube.com/embed/FnzUcGg60Qk" allowfullscreen></iframe>
```

Comment:
May be add alt description for the embedded video.

Reply:
I believe the title is enough here